VoodooInput/VoodooInputSimulator/VoodooInputSimulatorDevice.cpp

```diff
@@ -35,6 +35,8 @@ void VoodooInputSimulatorDevice::constructReportGated(const VoodooInputEvent& mu
     if (!ready_for_reports)
         return;
     
+    IOLog("VoodooInput got input, count: %d\n", multitouch_event.contact_count);
```

Comment:
Is this log necessary in the release version?

Reply:
No it's not, good catch as I was using that to debug.

Reply:
Fixed it in master.